src/navsat_transform.cpp

```diff
@@ -79,12 +80,13 @@ namespace RobotLocalization
     // Load the parameters we need
     nh_priv.getParam("magnetic_declination_radians", magnetic_declination_);
     nh_priv.param("yaw_offset", yaw_offset_, 0.0);
-    nh_priv.param("broadcast_utm_transform", broadcast_utm_transform_, false);
-    nh_priv.param("broadcast_utm_transform_as_parent_frame", broadcast_utm_transform_as_parent_frame_, false);
+    nh_priv.param("broadcast_utm_transform", broadcast_cartesian_transform_, false);
```

Comment:
Let's add two new parameters for `broadcast_cartesian_transform` and `broadcast_cartesian_transform_as_parent_frame` as well. Maybe check if the user set `broadcast_utm_transform` and issue a deprecation warning?

Reply:
I made this change and tested it locally, but I would like a second pair of eyes on it. Basically, if the old parameter is set, it issues a warning telling the user to use the new parameter. The value of the old parameters is not used, so they will revert to the defaults (`false`) for any user once they receive this update.